score.cpp

```diff
@@ -0,0 +1,124 @@
+#include <iostream>
+#include "score.h"
+
+
+Score::Score(){
+    //bestScore = 0;
+    currentScore = 0;
+    setScoreFile();
+    uploadBestScore();
+}
+
+Score::Score(std::string specialPath){
+    //bestScore = 0;
+    currentScore = 0;
+
+
+    std::filesystem::path spcPath = specialPath;
+    if(std::filesystem::exists(spcPath)){
+        std::filesystem::create_directories(spcPath / "data");
+        pathToFile = spcPath / "data/scores.txt";
+        std::cout << "Path to score file: " << pathToFile << std::endl;
+    }else{
+        setScoreFile();
+    }
+
+    uploadBestScore();
+}
+
+void Score::setScoreFile(){
+    try{
+        std::filesystem::path pathToRun = std::filesystem::current_path();
```

Comment:
Have you fixed the issue when calling the program from different directories will create the file in a different places?

Reply:
yes, but only if calling the program from project directories :(